src/app/beer_garden/api/http/base_handler.py

```diff
@@ -250,3 +251,29 @@ def schema_validated_body(self, schema: Type[Schema]) -> dict:
             return schema(strict=True).load(self.request_body).data
         except MarshmallowValidationError:
             raise BadRequest
+
+    def validate_or_raise(
+        self, schema: Type[Schema], arg: Any, method: str = "loads", many: bool = False
+    ):
+        """Apply a provided schema to an argument and raise an exception if it fails
+        to validate.
+
+        Args:
+            schema: A schema derived from a marshmallow Schema that the argment
+                will be validated against
+            arg: The data to validate
+            method: The method to call; one of `load`, `loads`, `dump` or `dumps`
```

Comment:
Converted to the more typical "from_string" arg.

Reply:
Yeah, I like that better.  Maybe `load_or_raise` would be a better name for this now?  Otherwise it still feels weird to call a method named "validate" that is in fact doing a load and returning an object.

Also, it looks like you docstring needs to be updated to reflect the changes.